### DISCUSSION TITLE
Link Update Request - Webhook Instructions for Teams

### DISCUSSION THREAD

**Discussion by lizoron** [QualiTorque/qualitorque.github.io#143] [Documentation requests] [answered]
Hey Steven,

This is regarding this article: https://docs.qtorque.io/admin-guide/notifications

"For Teams, see https://techcommunity.microsoft.com/t5/microsoft-365-pnp-blog/how-to-configure-and-use-incoming-webhooks-in-microsoft-teams/ba-p/2051118."

The support team and myself are unable to access this link above and requesting access does not work. So, I have another suggestion to replace it with:

https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook

I used this article for setting it up myself, so it seems to be clear to use. Let me know if you have any questions.

**Comment by TomerAdmon** [comment]
Thanks for the feedback @lizoron !
We will work on the docs improvment
@Shnitselon

**Comment by Shnitselon** [comment, marked as answer]
@lizoron Updated. Great feedback. Clear and straight to the point
Thanks a lot
